### PR TITLE
Add missing Context funcs to Driver interface

### PR DESCRIPTION
**Pull request by skateinmars**
Also bumps dependencies

### CODE REVIEW

File: sqalx.go [hunk 1]
```diff
@@ -49,13 +50,19 @@ type Driver interface {
...
 	NamedExec(query string, arg interface{}) (sql.Result, error)
+	NamedExecContext(ctx context.Context, query string, arg interface{}) (sql.Result, error)
 	NamedQuery(query string, arg interface{}) (*sqlx.Rows, error)
 	PrepareNamed(query string) (*sqlx.NamedStmt, error)
+	PrepareNamedContext(ctx context.Context, query string) (*sqlx.NamedStmt, error)
 	Preparex(query string) (*sqlx.Stmt, error)
+	PreparexContext(ctx context.Context, query string) (*sqlx.Stmt, error)
 	Rebind(query string) string
 	Select(dest interface{}, query string, args ...interface{}) error
+	SelectContext(ctx context.Context, dest interface{}, query string, args ...interface{}) error
```
Comment (rogpeppe) [inline sqalx.go]: FWIW this is missing `QueryRow` and `QueryRowContext` - if those are omitted deliberately, you might want to add a comment saying why.

For the record, I checked using this little bit of code: https://play.golang.org/p/mFtW7tIa68T

Comment (skateinmars) [reply sqalx.go]: Added!

**Comment by skateinmars** [comment]
@asdine or @rogpeppe can I let one of you merge and cut a new release?

File: sqalx.go [hunk 2]
```diff
@@ -49,13 +50,21 @@ type Driver interface {
...
+	QueryRow(string, ...interface{}) *sql.Row
```
Comment (rogpeppe) [inline sqalx.go]: Maybe good to preserve alphabetical order of methods here.

Comment (skateinmars) [reply sqalx.go]: :heavy_check_mark:

**Comment by skateinmars** [comment]
:up:

**Comment by jhchabran** [comment]
@skateinmars 

> LGTM with one trivial additional thought, thanks! 

The LGTM here means you can merge it!

**Comment by skateinmars** [comment]
@rogpeppe can I let you cut a new release? :pray:

**Pull request merged**